packages/react-components/src/molecules/CtaCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { Link, Card, Paragraph } from '../atoms';
+import {
+  mobileScreen,
+  perRem,
+  vminLinearCalcClamped,
+  tabletScreen,
+  smallDesktopScreen,
+} from '../pixels';
+
+const getInTouchStyles = css({
+  display: 'grid',
+  gridTemplateColumns: 'auto',
+  alignItems: 'center',
+  paddingTop: `${12 / perRem}em`,
+  paddingLeft: `${vminLinearCalcClamped(
+    mobileScreen,
+    24,
+    tabletScreen,
+    36,
+    'px',
+  )}`,
+  paddingRight: `${vminLinearCalcClamped(
+    mobileScreen,
+    24,
+    tabletScreen,
+    36,
+    'px',
+  )}`,
+  paddingBottom: `${vminLinearCalcClamped(
+    mobileScreen,
+    18,
+    tabletScreen,
+    24,
+    'px',
+  )}`,
+  [`@media (min-width: ${smallDesktopScreen.min}px)`]: {
+    gridTemplateColumns: 'auto min-content',
```

Comment:
```suggestion
    gridTemplateColumns: 'auto max-content',
```
I think our button probably has the same min-content as max-content because of `white-space: pre`, but if it ever doesn't because we need wrapping text in buttons, we should probably give this the space it wants